rainbow-me/browser-extension
# fix: browser translation feature causes password field to crash (seed phrase page)

Fixes BX-1480
Figma link (if any):

## What changed (plus any additional context for devs)

When you use `<></>` characters in JSX the browser translation feature is trying to manipulate the DOM in React which causes React Router to crash / throw an error.

<img width="854" alt="image" src="https://github.com/rainbow-me/browser-extension/assets/53529533/e9641c14-6b66-426c-9368-0eb59e7dc078">

## Screen recordings / screenshots

https://github.com/rainbow-me/browser-extension/assets/53529533/c5048266-fa35-4810-a52b-02f01c04b61d

## What to test

- Enable browser translation feature for any language
- Open a new instance of rainbow browser extension and try to create your first wallet while using browser translation feature
- Make sure whenever you type in the password you don't suddenly see a crash
